external-crates/move/crates/move-vm-runtime/src/interpreter.rs

```diff
@@ -167,23 +167,10 @@ impl Interpreter {
         ty_args: Vec<Type>,
         args: Vec<Value>,
     ) -> VMResult<Vec<Value>> {
-        let mut locals = Locals::new(function.local_count());
-        for (i, value) in args.into_iter().enumerate() {
-            locals
-                .store_loc(
-                    i,
-                    value,
-                    loader
-                        .vm_config()
-                        .enable_invariant_violation_check_in_swap_loc,
```

Comment:
This is the important bit. 
But this is now execution versioned, so it should be fine

Reply:
Can we also remove this from `store_local`

Reply:
We sadly cannot, because the version cuts rely on this API. That will have to change when we swap out the value representation, though, so I can come back and do this then.